### ISSUE TITLE
Basic usage failing import

### ISSUE THREAD

**Issue by andrei-gheorghiu** [TriDiamond/vuejs-countdown-timer#16]
Please see following sandbox: https://codesandbox.io/s/happy-meninsky-jf73z?file=/src/main.js

For some reason, the `import` is failing and component is not usable. 

Any suggestion?

**Comment by bennyxguo** [comment]
I fork it and test it, it's working fine: https://codesandbox.io/s/solitary-architecture-cy1fn

**Comment by andrei-gheorghiu** [comment]
Neither your fork nor what I posted work for me:
![image](https://user-images.githubusercontent.com/7866021/115620321-ee5d3700-a2ec-11eb-9755-71e80af63c72.png)

Getting a similar error on local. The very idea we're getting something entirely different out of the sane codesandbox is somewhat unsettling. What could cause it to behave differently? I'm using a recent Chrome version `Version 89.0.4389.128 (Official Build) (64-bit)`, on a windows 10 machine.

I also just tested it in incognito, to make sure it's not a cache issue. I'm seeing the same. Also tested in Firefox.  
Are you saying you're unable to repro?

**Comment by andrei-gheorghiu** [comment]
> I fork it and test it, it's working fine

I have a curiosity: why are you claiming something that is clearly not true? 
Your fork and what I posted have different package versions for multiple packages and a slightly different folder structure (I deleted some unnecessary stuff from mine, before posting it, while yours still has it).

So you clearly haven't forked mine. You created a new Vue sandbox and copied bits from mine. Which is perfectly fine, except for the part where you are not telling the truth. 

And why?
What are you trying to imply? That the problem doesn't exist? That it happens randomly? How could anyone trust you when you are not telling the truth about your own actions?
